<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Marie Saitou" />

<meta name="date" content="2022-10-09" />

<title>Genome Browser 2022</title>

<script src="site_libs/header-attrs-2.14/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Bio322</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/mariesaitou/Bio322">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Genome Browser 2022</h1>
<h4 class="author">Marie Saitou</h4>
<h4 class="date">10/09/2022</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-10-10
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 2
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Bio322/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcommariesaitouBio322tree3d7a0e92d43b9d31e321ca77ce93b949fd5201d3targetblank3d7a0e9a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/mariesaitou/Bio322/tree/3d7a0e92d43b9d31e321ca77ce93b949fd5201d3" target="_blank">3d7a0e9</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcommariesaitouBio322tree3d7a0e92d43b9d31e321ca77ce93b949fd5201d3targetblank3d7a0e9a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/mariesaitou/Bio322/tree/3d7a0e92d43b9d31e321ca77ce93b949fd5201d3" target="_blank">3d7a0e9</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/.DS_Store

Untracked files:
    Untracked:  210922_genome expression_epigenetics.pptx
    Untracked:  210927.module1.2.RNAseq_on_Galaxy.pdf
    Untracked:  220317_Advanced topics in genomics.docx
    Untracked:  220317_Advanced topics in genomics_MarieSai.docx
    Untracked:  BIO322_Teaching plan BIO322 2021.docx
    Untracked:  Bio322.09132021.pdf
    Untracked:  Bio322.09132021.pptx
    Untracked:  Bio322.09152021.backup.pptx
    Untracked:  Bio322.09152021.pdf
    Untracked:  Bio322.09152021.pptx
    Untracked:  Bio322.09202021.pdf
    Untracked:  Bio322.09202021.pptx
    Untracked:  Bio322.09272021.pptx
    Untracked:  Bio322.09272021/
    Untracked:  Bio322scRNAseq.tsv
    Untracked:  Bio_322.docx
    Untracked:  Bio_322.pdf
    Untracked:  Galaxy1-[intestinalData.tsv].tabular
    Untracked:  Galaxy2.txt
    Untracked:  Group.csv
    Untracked:  analysis/Evolution_for_lab.Rmd
    Untracked:  analysis/_site/
    Untracked:  analysis/tutorial.RNAseq.foradults.xlsx
    Untracked:  bio322.xlsx
    Untracked:  bio322_2022.pptx
    Untracked:  chr15_inversion-v1.0.0.zip
    Untracked:  gene_regulation_bio322_2022.pptx
    Untracked:  gwassim.txt
    Untracked:  intestinalData.tsv
    Untracked:  main_workflow.ga
    Untracked:  markdown_test/
    Untracked:  mouse_intestine_scRNAseq.txt
    Untracked:  oharring-chr15_inversion-9615456/
    Untracked:  science.abg0718_data_s1_to_s8.zip
    Untracked:  science.abg0718_data_s1_to_s8/
    Untracked:  scrna_tenx.ga

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/2022_genome.function2.Rmd</code>)
and HTML (<code>docs/2022_genome.function2.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mariesaitou/Bio322/blob/3d7a0e92d43b9d31e321ca77ce93b949fd5201d3/analysis/2022_genome.function2.Rmd" target="_blank">3d7a0e9</a>
</td>
<td>
mariesaitou
</td>
<td>
2022-10-10
</td>
<td>
wflow_publish("analysis/2022_genome.function2.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mariesaitou/Bio322/82d7bc20c0dfc3d17e4b6d630e09b947f8d0358f/docs/2022_genome.function2.html" target="_blank">82d7bc2</a>
</td>
<td>
mariesaitou
</td>
<td>
2022-10-10
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mariesaitou/Bio322/blob/d99cec74171253e71e3e89109ab721298e5e4f3c/analysis/2022_genome.function2.Rmd" target="_blank">d99cec7</a>
</td>
<td>
mariesaitou
</td>
<td>
2022-10-10
</td>
<td>
wflow_publish("analysis/2022_genome.function2.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mariesaitou/Bio322/1378cc548e5894e37379e7efa63f4a83004e46eb/docs/2022_genome.function2.html" target="_blank">1378cc5</a>
</td>
<td>
mariesaitou
</td>
<td>
2022-10-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mariesaitou/Bio322/03bf4d7eb5a35800884911d440b072f45fb4aa4b/docs/2022_genome.function2.html" target="_blank">03bf4d7</a>
</td>
<td>
mariesaitou
</td>
<td>
2022-10-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mariesaitou/Bio322/blob/331672ba18f54f581149391a201bd47d8c85c814/analysis/2022_genome.function2.Rmd" target="_blank">331672b</a>
</td>
<td>
mariesaitou
</td>
<td>
2022-10-09
</td>
<td>
wflow_publish("analysis/2022_genome.function2.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mariesaitou/Bio322/e43ff3208fa23c87e4829df720c8f10d3eaa7070/docs/2022_genome.function2.html" target="_blank">e43ff32</a>
</td>
<td>
mariesaitou
</td>
<td>
2022-10-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mariesaitou/Bio322/blob/861f5c85418a629c5d75963fbe126f01c225f1f8/analysis/2022_genome.function2.Rmd" target="_blank">861f5c8</a>
</td>
<td>
mariesaitou
</td>
<td>
2022-10-09
</td>
<td>
wflow_publish("analysis/2022_genome.function2.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mariesaitou/Bio322/a0cae876845b044b06958ab4c0cf28669d0c6de2/docs/2022_genome.function2.html" target="_blank">a0cae87</a>
</td>
<td>
mariesaitou
</td>
<td>
2022-10-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mariesaitou/Bio322/blob/14a66d02545e1d54d7320434fda3e32620bc2fbe/analysis/2022_genome.function2.Rmd" target="_blank">14a66d0</a>
</td>
<td>
mariesaitou
</td>
<td>
2022-10-09
</td>
<td>
wflow_publish("analysis/2022_genome.function2.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mariesaitou/Bio322/e507e70b56d9056795e2783dd4baa7a8f25c95c5/docs/2022_genome.function2.html" target="_blank">e507e70</a>
</td>
<td>
mariesaitou
</td>
<td>
2022-10-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mariesaitou/Bio322/0c87f6266de0cb30ad100797571cead7a9087a1c/docs/2022_genome.function2.html" target="_blank">0c87f62</a>
</td>
<td>
mariesaitou
</td>
<td>
2022-10-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mariesaitou/Bio322/blob/59378af9f996eb6bd34968959a111c22cd6dd157/analysis/2022_genome.function2.Rmd" target="_blank">59378af</a>
</td>
<td>
mariesaitou
</td>
<td>
2022-10-09
</td>
<td>
wflow_publish("analysis/2022_genome.function2.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mariesaitou/Bio322/664b5c6792f3128613d40d6af2b5f24965b62964/docs/2022_genome.function2.html" target="_blank">664b5c6</a>
</td>
<td>
mariesaitou
</td>
<td>
2022-10-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mariesaitou/Bio322/blob/b56d9ff878a21eb8dfb24d1afaa85bfa6fce4706/analysis/2022_genome.function2.Rmd" target="_blank">b56d9ff</a>
</td>
<td>
mariesaitou
</td>
<td>
2022-10-09
</td>
<td>
wflow_publish("analysis/2022_genome.function2.Rmd")
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="genome-variation-and-function-2" class="section level1">
<h1>Genome variation and function 2</h1>
<p>Goal: Today, we learn about how to analyze genetic variants and its
association with phenotype.</p>
<ol style="list-style-type: decimal">
<li><p>Genome wide association study - theory</p></li>
<li><p>Genome wide association study - browse real data</p></li>
</ol>
<p>Relevant review papers if you want to further learn:</p>
<p><a
href="https://www.nature.com/articles/s41576-019-0127-1">General</a></p>
<p>Depends on your interest:</p>
<p><a href="https://www.nature.com/articles/nrg.2017.82">Plant</a></p>
<p><a
href="https://www.nature.com/articles/s41576-018-0082-2">Animal</a></p>
<p><a
href="https://www.nature.com/articles/s41576-020-0227-y">Fish</a></p>
<div id="lecture-part" class="section level2">
<h2>1. Lecture part</h2>
<p><a
href="https://github.com/mariesaitou/Bio322/blob/master/docs/assets/Bio322.2022/bio322_2022/Bio322_module2_functional_genomics2.pdf">PDF</a></p>
<p>Prologue - Current topics in genomics</p>
<p>Human genome sequence</p>
<p>Nobel prize 2022</p>
<p>We will learn about:</p>
<p>Methods and significance of functional genomics</p>
<p>Challenges of genomics</p>
</div>
<div id="hands-on-tasks" class="section level2">
<h2>2. Hands-on tasks</h2>
<p>Now, we will investigate real GWAS data at GWAS ATLAS, which concains
4,756 human GWAS across 3,302 traits.</p>
<p>Let’s explore the human GWAS database <a
href="https://atlas.ctglab.nl/">GWAS ATLAS</a>. In the tutorial, I did
not explain everything, so please explore it yourself and get
familiarised with it.</p>
</div>
<div id="case-study-1" class="section level2">
<h2>Case study 1</h2>
<p>Go to “Browse GWAS” in the top, black bar. And search for the trait
about hair morphology and see the results.</p>
<p><img
src="https://mariesaitou.github.io/Bio322/assets/Bio322.2022/hair.png" /></p>
</div>
<div id="questions-1" class="section level2">
<h2>Questions 1</h2>
<p>Q1-1. Examine the Manhattan plot. Where do you see the peak? How can
we interpret this?</p>
<p>Q1-2. See the left table. How many individuals were investigated?</p>
<p>Q1-3. How many variants were investigated?</p>
<p>Q1-4. See the table under the Manhattan plot with gene annotations.
What is the ID of top-associated SNP? (SNP ID: rsXXXXX)</p>
<p>Q2-5. Go to <a href="https://popgen.uchicago.edu/ggv/">GGV map
browser</a> and enter the SNP ID above to see the global distribution of
this variant. “A” is the straight hair allele, and “G” is the curly hair
allele. What can we estimate from this map? Why did it happen?</p>
<div id="result-1" class="section level3">
<h3>Result 1</h3>
<details>
<summary>
<strong>Click to display</strong>
</summary>
<p><a href="https://atlas.ctglab.nl/traitDB/4023"
class="uri">https://atlas.ctglab.nl/traitDB/4023</a></p>
<p>A1-1. There is one big peak at chromosome 2. It is plausible that one
genetic factor with very strong effect is associated with this
trait.</p>
<p>A1-2. 4878 individuals</p>
<p>A1-3. 560921 SNPs</p>
<p>A1-4. rs260643</p>
<p>A1-5. Straight hair type is thought to be adaptive in East Asia in
human evolution. Why? There are a lot of theory but there is not a
concrete consensus yet.</p>
<p><a
href="https://www.sciencedirect.com/science/article/pii/S0092867413000676">if
you are interested</a> In this paper, they generated a knock-in mouse
model to investigate the function of hair-morphology altering variant.
In the mouce model, they observed increased hair thickness, but also
change the morphology of mammary and eccrine glands.</p>
</details>
</div>
</div>
<div id="case-study-2" class="section level2">
<h2>Case study 2</h2>
<p>Go to “Browse GWAS” in the top, black bar. And search for the trait
about eyesight and see the results.</p>
<p><img
src="https://mariesaitou.github.io/Bio322/assets/Bio322.2022/eye.png" /></p>
</div>
<div id="questions-2" class="section level2">
<h2>Questions 2</h2>
<p>Q2-1. Examine the Manhattan plot. Where do you see the peak? How can
we interpret this?</p>
<p>Q2-2. See the left table. How many individuals were investigated?</p>
<p>Q2-3. How many variants were investigated?</p>
<p>Q2-4. See the bottom Manhattan plot with gene annotations. What is
the top-associated gene?</p>
<p>Q2-5. (Discussion topic) - Do some literature search about the
top-associated genes and assume how this gene is associated with the
trait, “Reason for glasses/contact lenses: For short-sightedness”. To
further understand the mechanism, what experiment would you plan?</p>
<div id="result-2" class="section level3">
<h3>Result 2</h3>
<details>
<summary>
<strong>Click to display</strong>
</summary>
<p><a href="https://atlas.ctglab.nl/traitDB/3539"
class="uri">https://atlas.ctglab.nl/traitDB/3539</a></p>
<p>A2-1. There are multiple peaks, in the chromosomes 1, 2, 4, 6, 8, 10,
15… The obserbation implies that multiple loci are associated with this
trait (The trait is polygenic).</p>
<p>A2-2. 78647 samples (See the left box, “N”)</p>
<p>A2-3. 9223534 SNPs (See the left box, “Nsnps”)</p>
<p>A2-4. PRSS56 (the dot with lowest P-value = highest -log10P
value)</p>
<p>A2-5. Put what you have discussed on CANVAS “Discussion” with the
full names of participants who contributed to the discussion.</p>
</details>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
