<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Marie Saitou" />


<title>Genome Browser 2022</title>

<script src="site_libs/header-attrs-2.14/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Bio322</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/mariesaitou/Bio322">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Genome Browser 2022</h1>
<h4 class="author">Marie Saitou</h4>
<h4 class="date">4/23/2022</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-10-08
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 2
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Bio322/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcommariesaitouBio322treeb929869674294d31bcde264e6051d0cddf8c1336targetblankb929869a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/mariesaitou/Bio322/tree/b929869674294d31bcde264e6051d0cddf8c1336" target="_blank">b929869</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcommariesaitouBio322treeb929869674294d31bcde264e6051d0cddf8c1336targetblankb929869a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/mariesaitou/Bio322/tree/b929869674294d31bcde264e6051d0cddf8c1336" target="_blank">b929869</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/.DS_Store

Untracked files:
    Untracked:  210922_genome expression_epigenetics.pptx
    Untracked:  210927.module1.2.RNAseq_on_Galaxy.pdf
    Untracked:  220317_Advanced topics in genomics.docx
    Untracked:  220317_Advanced topics in genomics_MarieSai.docx
    Untracked:  BIO322_Teaching plan BIO322 2021.docx
    Untracked:  Bio322.09132021.pdf
    Untracked:  Bio322.09132021.pptx
    Untracked:  Bio322.09152021.backup.pptx
    Untracked:  Bio322.09152021.pdf
    Untracked:  Bio322.09152021.pptx
    Untracked:  Bio322.09202021.pdf
    Untracked:  Bio322.09202021.pptx
    Untracked:  Bio322.09272021.pptx
    Untracked:  Bio322.09272021/
    Untracked:  Bio322scRNAseq.tsv
    Untracked:  Galaxy1-[intestinalData.tsv].tabular
    Untracked:  Galaxy2.txt
    Untracked:  Group.csv
    Untracked:  analysis/2022_genome.function2.Rmd
    Untracked:  analysis/Evolution_for_lab.Rmd
    Untracked:  analysis/_site/
    Untracked:  analysis/genomebrowser_for_lab.Rmd
    Untracked:  analysis/tutorial.RNAseq.foradults.xlsx
    Untracked:  bio322.xlsx
    Untracked:  bio322_2022.pptx
    Untracked:  chr15_inversion-v1.0.0.zip
    Untracked:  gene_regulation_bio322_2022.pptx
    Untracked:  gwassim.txt
    Untracked:  intestinalData.tsv
    Untracked:  main_workflow.ga
    Untracked:  markdown_test/
    Untracked:  mouse_intestine_scRNAseq.txt
    Untracked:  oharring-chr15_inversion-9615456/
    Untracked:  science.abg0718_data_s1_to_s8.zip
    Untracked:  science.abg0718_data_s1_to_s8/
    Untracked:  scrna_tenx.ga

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/2022_genome.function1.Rmd</code>)
and HTML (<code>docs/2022_genome.function1.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mariesaitou/Bio322/blob/b929869674294d31bcde264e6051d0cddf8c1336/analysis/2022_genome.function1.Rmd" target="_blank">b929869</a>
</td>
<td>
mariesaitou
</td>
<td>
2022-10-08
</td>
<td>
wflow_publish("analysis/2022_genome.function1.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mariesaitou/Bio322/ab351d7482b0ca4e92cc4c9c626a5bcaf26df61c/docs/2022_genome.function1.html" target="_blank">ab351d7</a>
</td>
<td>
mariesaitou
</td>
<td>
2022-10-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mariesaitou/Bio322/blob/c9b08bd19b25b1df8dacc11e7bd08902c104c173/analysis/2022_genome.function1.Rmd" target="_blank">c9b08bd</a>
</td>
<td>
mariesaitou
</td>
<td>
2022-10-04
</td>
<td>
wflow_publish("analysis/2022_genome.function1.Rmd")
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="genome-variation-and-function-1" class="section level1">
<h1>Genome variation and function 1</h1>
<p>Goal: Today, we learn about how to analyze genetic variants between
individuals of the same species.</p>
<ol style="list-style-type: decimal">
<li><p>How to read a “variant call format” (VCF) file</p></li>
<li><p>How to investigate the phenotypic effect of variants</p></li>
</ol>
</div>
<div id="variant-call-format" class="section level1">
<h1>1 Variant call format</h1>
<div id="interpret-a-vcf-file" class="section level2">
<h2>Interpret a VCF file</h2>
<p><img src="assets/Bio322.2022/Slide2.png" style="width:80.0%" /></p>
<p><a
href="https://towardsdatascience.com/simple-convolution-neural-network-for-genomic-variant-calling-with-tensorflow-c085dbc2026f">ref:
Variant calling format: VCF file</a></p>
<p><img src="assets/Bio322.2022/Slide3.png" style="width:80.0%" /></p>
<p><a href="https://en.wikipedia.org/wiki/Variant_Call_Format">ref: How
to read a VCF file</a></p>
<p>Header with “#” — information field.</p>
<p>CHROM: chromosome of the variant location</p>
<p>POS: position of the variant</p>
<p>ID: ID of each variant to make it easy to manage</p>
<p>REF: reference allele (the same type as the reference genome)</p>
<p>ALT: alternative allele (the mutant type which is different from the
reference genome)</p>
<p>QUAL FILTER INFO FORMAT: variant calling quality and other detail of
the variant</p>
<p>Samples… Genotype. “0” means “reference type”, “1” means “alternative
type”</p>
<div id="example" class="section level3">
<h3>Example</h3>
<ol style="list-style-type: decimal">
<li><p>The first variant is at chromosome 2, 81170 base.</p></li>
<li><p>The first variant’s reference allele is C, alternative allele is
T.</p></li>
<li><p>For The first variant, Sample1’s genotype is C/C, Sample2’s is
C/C, Sample3’s is C/T.</p></li>
</ol>
</div>
</div>
</div>
<div id="the-effect-of-genetic-variants" class="section level1">
<h1>2 The effect of genetic variants</h1>
<p><img src="assets/Bio322.2022/Slide4.png" style="width:80.0%" /></p>
<p>You found a genetic variant in your sequenced individual. You want to
investigate the potential effect of the variants.</p>
<p>Assume that you found the following deletion polymorphism at in a
rabbit genome.</p>
<p>12: 107,236,296-107,236,969</p>
<p><a href="https://www.ensembl.org/Tools/VEP">Go to the Variant Effect
Predictor</a></p>
<p>Input the following information:</p>
<p><em>Species - rabbit (Oryctolagus_cuniculus)</em></p>
<p><em>Variant - 12 107236296 107236969 DEL + deletion1</em></p>
<p>the variant format, left to right … chromosome, starting point,
ending point, kind, strand, variant ID (you can name it as you like)</p>
<p>Variants.</p>
<p><img src="assets/2022_browser/image-04-25-18.48.31.png"
style="width:80.0%" /></p>
<p>You can also find various acceptable input formats <a
href="https://www.ensembl.org/info/docs/tools/vep/vep_formats.html">here</a></p>
<p><img src="assets/2022_browser/image-04-25-18.53.12.png"
style="width:80.0%" /></p>
<p>The result shows that the deletion is affecting a gene, SLC35F1, by
causing coding sequence variant, splice variant, feature truncation and
intron variant. You can modify the result table by clicking “show/hide
columns”</p>
<p>Now you found that the variant is overlapping a gene, and you want to
know the function of the gene.</p>
<p>Click “ENSOCUG00000015307” to know more about this gene.</p>
<p>Sometimes, gene function information is not well described in
non-model species. But in other species such as mouse, the homologous
gene function may have been investigated.</p>
<p>In the gene page, you can investigate its evolutionary history and
potential function.</p>
<p><img src="assets/2022_browser/image-04-25-19.10.05.png"
style="width:80.0%" /></p>
<div id="evolutionary-insights" class="section level2">
<h2>Evolutionary insights</h2>
<p>Let’s explore “gene gain/loss tree” “Onthologies”and “Phenotypes”</p>
<p>Gene gain/loss tree</p>
<p><img src="assets/2022_browser/image-04-25-19.11.48.png"
style="width:80.0%" /></p>
<p>Most of mammalian species have two copies of this gene.</p>
<p><img src="assets/2022_browser/image-04-25-19.12.04.png"
style="width:80.0%" /></p>
<p>See what is happening in fish!</p>
<p>Gene Ontology <img src="assets/2022_browser/image-04-25-19.12.34.png"
style="width:80.0%" /> The gene is related to “transmembrane
transtport”.</p>
<p><img src="assets/2022_browser/image-04-25-19.13.13.png"
style="width:80.0%" /></p>
<p>Phenotypes. There is no phenotype information for this gene in
rabbits, but there are some reports in mouse and rat, such as leukocyte
cell number etc.</p>
</div>
</div>
<div id="hands-on-part" class="section level1">
<h1>3 Hands-on part</h1>
<p>Now, you have <a href="assets/2022_browser/sample.vcf">a VCF file -
click here</a> of dogs (Canis_lupus_familiaris).</p>
<p>Please explore the genetic variation in the file with the tools
above.</p>
<div id="questions-1" class="section level2">
<h2>Questions 1</h2>
<p>Q1-1. How many variants are there in the VCF file?</p>
<p>Q1-2. How many individuals are there in the VCF file?</p>
<p>Q1-3. What is the genotype of sample 1 for variant 1?</p>
<p>Q1-4. What is the reference and alternative alleles of variant 2?</p>
<p>Q1-5. Please describe the genotype frequency of variant 3.</p>
<p>Q1-6. Please describe the allele frequency of variant 15.</p>
<div id="result1" class="section level3">
<h3>Result1</h3>
<details>
<summary>
<strong>Click to display</strong>
</summary>
<p>A1-1. 15 variants</p>
<p>A1-2. 18 samples</p>
<p>A1-3. GG</p>
<p>A1-4. Reference: T. Alternative:
TAGACACTGTCTACCGAAATGTAGACACTGTCTACCGAAATG</p>
<p>A1-5.</p>
<p>There are sixteen 0|0, one 0|1 and one 1|1 individuals.</p>
<p>Each genotype frequency is:</p>
<p>G/G:0.056 (1 out of 18)</p>
<p>G/GGG:0.89 (16 out of 18)</p>
<p>GGG/GGG:0.056 (1 out of 18)</p>
<p>A1-6.</p>
<p>There are fifteen 0|0, two 0|1 and one 1|1 individuals.</p>
<p>There are 2 * 18 = 36 alleles (each individual has two alleles)</p>
<p>Each allele frequency is:</p>
<p>A allele: 0.89 (15 * 2 + 2 * 1 = 32, 32 our of 36)</p>
<p>G allele: 0.11 (2 * 1 + 1 * 2 = 4, 4 out of 36)</p>
</details>
</div>
</div>
<div id="questions-2" class="section level2">
<h2>Questions 2</h2>
<p>Q2-1 How many genes were overlapped with submitted variants?</p>
<p>Q2-2 How many variants are overlapped with intronic regions?</p>
<p>Q2-3 How many variants are known, and how many are not reported?</p>
<p>Q2-4 What kind of exonic variants are there?</p>
<p>Q2-5 What are the predicted amino acid changes by the “missense
variant” among them? Which exon is it?</p>
<div id="result2" class="section level3">
<h3>Result2</h3>
<details>
<summary>
<strong>Click to display</strong>
</summary>
<p>A2-1. Three (LCT, SPOPL, UBXN4)</p>
<p>A2-2. Ten variants</p>
<p>A2-3. Nine variants are already reported (existing), and six variants
are not reported (novel).</p>
<p>A2-4. inframe_insertion, synonymous_variant, frameshift_variant,
missense_variant</p>
<p>A2-5. Q -&gt; R (Glutamine -&gt; Arginine), the 8th of the 17
exons.</p>
</details>
</div>
</div>
<div id="questions-3" class="section level2">
<h2>Questions 3</h2>
<p>Q3-1 What is the “ontology” of the LCT gene?</p>
<p>Q3-2 What is the phenotype of the LCT-deficient rodents?</p>
<p>Q3-3 Do mammalian and non-mammalian species have the same number of
the LCT gene?</p>
<div id="result2-1" class="section level3">
<h3>Result2</h3>
<details>
<summary>
<strong>Click to display</strong>
</summary>
<p>A3-1</p>
<p>Molecular function: protein homodimerization activity</p>
<p>Cellular component: plasma membrane</p>
<p>Biological process: metabolic process</p>
<p>— LCT is located in membrane, and contributes to metabolic process
with its protein homodimerization activity.</p>
<p>Show/hide columns (1 hidden)</p>
<p>A3-2</p>
<p>Lactose Intolerance, Diarrhea, colitis etc.</p>
<p>A3-3</p>
<p>Most mammals have one LCT gene. Fish species have variable number of
LCT gene, such as eight genes in Bicolor damselfish.</p>
</details>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
