<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Marie Saitou" />


<title>Genome Browser 2022</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Bio322</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/mariesaitou/Bio322">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Genome Browser 2022</h1>
<h4 class="author">Marie Saitou</h4>
<h4 class="date">17/09/2022</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-10-17
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 2
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Bio322/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version
1.6.2). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcommariesaitouBio322tree1ba19cddb97c943882cf0486909fbcf19c5012cbtargetblank1ba19cda">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/mariesaitou/Bio322/tree/1ba19cddb97c943882cf0486909fbcf19c5012cb" target="_blank">1ba19cd</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcommariesaitouBio322tree1ba19cddb97c943882cf0486909fbcf19c5012cbtargetblank1ba19cda"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/mariesaitou/Bio322/tree/1ba19cddb97c943882cf0486909fbcf19c5012cb" target="_blank">1ba19cd</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  210922_genome expression_epigenetics.pptx
    Untracked:  210927.module1.2.RNAseq_on_Galaxy.pdf
    Untracked:  220317_Advanced topics in genomics.docx
    Untracked:  220317_Advanced topics in genomics_MarieSai.docx
    Untracked:  BIO322_Teaching plan BIO322 2021.docx
    Untracked:  Bio322.09132021.pdf
    Untracked:  Bio322.09132021.pptx
    Untracked:  Bio322.09152021.backup.pptx
    Untracked:  Bio322.09152021.pdf
    Untracked:  Bio322.09152021.pptx
    Untracked:  Bio322.09202021.pdf
    Untracked:  Bio322.09202021.pptx
    Untracked:  Bio322.09272021.pptx
    Untracked:  Bio322.09272021/
    Untracked:  Bio322scRNAseq.tsv
    Untracked:  Bio_322.docx
    Untracked:  Bio_322.pdf
    Untracked:  Galaxy1-[intestinalData.tsv].tabular
    Untracked:  Galaxy2.txt
    Untracked:  Group.csv
    Untracked:  analysis/Evolution_for_lab.Rmd
    Untracked:  analysis/_site/
    Untracked:  analysis/tutorial.RNAseq.foradults.xlsx
    Untracked:  bio322.xlsx
    Untracked:  bio322_2022.pptx
    Untracked:  chr15_inversion-v1.0.0.zip
    Untracked:  gene_regulation_bio322_2022.pptx
    Untracked:  gwassim.txt
    Untracked:  intestinalData.tsv
    Untracked:  main_workflow.ga
    Untracked:  markdown_test/
    Untracked:  mouse_intestine_scRNAseq.txt
    Untracked:  oharring-chr15_inversion-9615456/
    Untracked:  science.abg0718_data_s1_to_s8.zip
    Untracked:  science.abg0718_data_s1_to_s8/
    Untracked:  scrna_tenx.ga

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/2022_genome.function3.Rmd</code>)
and HTML (<code>docs/2022_genome.function3.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mariesaitou/Bio322/blob/1ba19cddb97c943882cf0486909fbcf19c5012cb/analysis/2022_genome.function3.Rmd" target="_blank">1ba19cd</a>
</td>
<td>
mariesaitou
</td>
<td>
2022-10-17
</td>
<td>
wflow_publish("analysis/2022_genome.function3.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mariesaitou/Bio322/93d8ce021f1e8b81de6a766544c10f33635468f5/docs/2022_genome.function3.html" target="_blank">93d8ce0</a>
</td>
<td>
mariesaitou
</td>
<td>
2022-10-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mariesaitou/Bio322/blob/6d520fb9d76629244274c4b4fac5c4e090822117/analysis/2022_genome.function3.Rmd" target="_blank">6d520fb</a>
</td>
<td>
mariesaitou
</td>
<td>
2022-10-17
</td>
<td>
wflow_publish("analysis/2022_genome.function3.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mariesaitou/Bio322/9896144efce4142b0eb2182886e318d68a3436ab/docs/2022_genome.function3.html" target="_blank">9896144</a>
</td>
<td>
mariesaitou
</td>
<td>
2022-10-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mariesaitou/Bio322/blob/a06fd2d5efbb66df2b0078a3145494b0e75a4292/analysis/2022_genome.function3.Rmd" target="_blank">a06fd2d</a>
</td>
<td>
mariesaitou
</td>
<td>
2022-10-16
</td>
<td>
wflow_publish("analysis/2022_genome.function3.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mariesaitou/Bio322/72a15b5e973e00003d30c05397517d7365920832/docs/2022_genome.function3.html" target="_blank">72a15b5</a>
</td>
<td>
mariesaitou
</td>
<td>
2022-10-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mariesaitou/Bio322/blob/8686188c071f0cb74618d525e3170ead715bdf9e/analysis/2022_genome.function3.Rmd" target="_blank">8686188</a>
</td>
<td>
mariesaitou
</td>
<td>
2022-10-16
</td>
<td>
wflow_publish("analysis/2022_genome.function3.Rmd")
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="genome-variation-and-function-3" class="section level1">
<h1>Genome variation and function 3</h1>
<p>Goal: Today, we learn about transcriptome, and its association with
genotype.</p>
<ol style="list-style-type: decimal">
<li><p>RNAseq and eQTL - theory</p></li>
<li><p>RNAseq and eQTL - browse real data</p></li>
</ol>
<p>Relevant review papers if you want to further learn:</p>
<p><a href="https://www.nature.com/articles/nrg3891">eQTL</a></p>
<div id="lecture-part" class="section level2">
<h2>1. Lecture part</h2>
<p><a
href="https://github.com/mariesaitou/Bio322/blob/master/docs/assets/Bio322.2022/bio322_2022/Bio322_module2_functional_genomics3.pdf">PDF</a></p>
</div>
<div id="hands-on-tasks" class="section level2">
<h2>2. Hands-on tasks</h2>
<p>Now, we will investigate real transcriptome data at GTEx Portal,
which concains human RNAseq date from 948 donors across 54 tissues.</p>
<p>Let’s explore the human ranscriptomet database <a
href="https://gtexportal.org/home/">GTEx Portal</a>. In the tutorial, I
did not explain everything, so please explore it yourself and get
familiarised with it.</p>
<p>Go to “GTEx” in the top, black bar. And search for the gene “ADH1C
(Alcohol Dehydrogenase 1C)” and see the results.</p>
</div>
<div id="questions-1" class="section level2">
<h2>Questions 1</h2>
<p><strong>Q1-1.</strong> See “Bulk tissue gene expression”. Where is
the ADH1C gene highly expressed?</p>
<p><strong>Q1-2.</strong> Search for the ADH1A and ADH1B as well. Are
their expression pattern different from ADH1C? How does it happen?
(Remember Simen’s module1)</p>
<p><strong>Q1-3.</strong> Go back to “ADH1C”. Do you see any eQTL SNPs
in “Significant Single-Tissue eQTLs for ADH1C”? What is the top eQTL SNP
with the lowest P-value and which tissue is it? (You can sort the
variants by clicking “p-value”.)</p>
<p>Open the “eQTL violin plot”. Which genotype is associated with the
higher expression of ADH1C?</p>
<p><strong>Q1-4.</strong> Can we conclude that the eQTL (SNP-gene
combination) in that tissue in Q1-3 is the most influential for ADH1C
function biologically? If not, why?</p>
<details>
<summary>
<strong>Hint</strong>
</summary>
<p>The violin plot is normalized and does not reflect the absolute
expression in that tissue.</p>
<p><a
href="https://gtexportal.org/home/methods#staticTextAnalysisMethods">Further
reading</a></p>
<p>Gene expression values for all samples from a given tissue were
normalized using the following procedure:</p>
<p>Genes were selected based on expression thresholds of &gt;0.1 TPM in
at least 20% of samples and ≥6 reads in at least 20% of samples.</p>
<p>Expression values were normalized between samples using TMM as
implemented in edgeR (Robinson &amp; Oshlack, Genome Biology, 2010
).</p>
<p>For each gene, expression values were normalized across samples using
an inverse normal transform.</p>
</details>
<p><strong>Q1-5.</strong> Search for “liver” in the “Significant
Single-Tissue eQTLs for ADH1C”. Let’s take a look at the violin plot of
the top variant, rs2173201. Which genotype is associated with the high
expression of ADH1C? Can we conclude that the eQTL (SNP-gene
combination) is the most influential for ADH1C function biologically? If
not, why?</p>
<p><strong>Q1-6.</strong> Go to <a
href="https://atlas.ctglab.nl/PheWAS">PheWAS</a> and enter the SNP ID
above (rs2173201) to see if this variant is associated with any traits.
What is the top traits with the lowest P-value? EA (effect allele) means
allele increases the risk of the phenotype and NEA (non-effect allele)
means other alleles. Which allele (nucleotide) is associated with the
risk-increasing effect of the top trait?</p>
<p><strong>Q1-7.</strong> Discuss how the lower expression of ADH1C
(Alcohol Dehydrogenase 1C) is associated with higher alcohol intake (you
don’t have to submit it).</p>
<p><a
href="https://pubs.niaaa.nih.gov/publications/aa72/aa72.htm">Reading
material</a></p>
<div id="result-1" class="section level3">
<h3>Result 1</h3>
<details>
<summary>
<strong>Click to display</strong>
</summary>
<p><a href="https://gtexportal.org/home/gene/ADH1C"
class="uri">https://gtexportal.org/home/gene/ADH1C</a></p>
<p><strong>A1-1.</strong> ADH1C is highly expressed in liver, but also
in colon and stomach.</p>
<p><strong>A1-2.</strong> ADH1A is mainly expressed in liver, while
ADH1B is highly expressed in various tissues.</p>
<p><strong>A1-3.</strong> chr4_99349450_C_T_b38 (rs283415), Adipose -
Subcutaneous</p>
<p><strong>A1-4.</strong> TT is associated with the higher ADH1C
expression in Adipose. However, The ADH1C expression in Adipose is much
lower (Median TPM = 14.23) compared to liver (Median TPM = 273.7), and
we cannot conclude that the eQTL in Adipose has the primary effect for
ADH1C function.</p>
<p><strong>A1-5.</strong> AA genotype is associated with the higher
ADH1C expression in liver. However, eQTL only indicates the statistical
association between genotypes and gene expression, and this does not
tell the biological mechanism regarding it.</p>
<p><strong>A1-6.</strong> C allele is associated with higher alcohol
intake and higher Alcohol dependency risk.</p>
</details>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
