<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Marie Saitou" />


<title>Bio322.ensembl1</title>

<script src="site_libs/header-attrs-2.10/header-attrs.js"></script>
<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Bio322</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/mariesaitou/Bio322">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Bio322.ensembl1</h1>
<h4 class="author">Marie Saitou</h4>
<h4 class="date">8/23/2021</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-09-10
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 2 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Bio322/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcommariesaitouBio322tree899b47036e97c4c0c8ca8fbf7b68eec386dde0c1targetblank899b470a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/mariesaitou/Bio322/tree/899b47036e97c4c0c8ca8fbf7b68eec386dde0c1" target="_blank">899b470</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcommariesaitouBio322tree899b47036e97c4c0c8ca8fbf7b68eec386dde0c1targetblank899b470a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/mariesaitou/Bio322/tree/899b47036e97c4c0c8ca8fbf7b68eec386dde0c1" target="_blank">899b470</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/ensembl1.rmd</code>) and HTML (<code>docs/ensembl1.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mariesaitou/Bio322/blob/899b47036e97c4c0c8ca8fbf7b68eec386dde0c1/analysis/ensembl1.rmd" target="_blank">899b470</a>
</td>
<td>
mariesaitou
</td>
<td>
2021-09-10
</td>
<td>
wflow_publish(“analysis/ensembl1.Rmd”)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mariesaitou/Bio322/f139bdf6f7e9878d0760e02c68866d6ce6fc6358/docs/ensembl1.html" target="_blank">f139bdf</a>
</td>
<td>
mariesaitou
</td>
<td>
2021-09-10
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mariesaitou/Bio322/f863fe09558b27f1b742bd22bafc65ac04bdbb8f/docs/ensembl1.html" target="_blank">f863fe0</a>
</td>
<td>
mariesaitou
</td>
<td>
2021-09-10
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mariesaitou/Bio322/c867c588abfd88da11f35f87429a240eb2c17031/docs/ensembl1.html" target="_blank">c867c58</a>
</td>
<td>
mariesaitou
</td>
<td>
2021-09-10
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mariesaitou/Bio322/blob/c41e75fc9f439003e78093f2388e284db41e53e1/analysis/ensembl1.rmd" target="_blank">c41e75f</a>
</td>
<td>
mariesaitou
</td>
<td>
2021-09-10
</td>
<td>
wflow_publish(“analysis/ensembl1.Rmd”)
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="introduction" class="section level2">
<h2>0. Introduction</h2>
<p>This is a set of introductory hands-on exercises to get to know more about the genomes of various animals with public databases. These exercises align the learning goals from module 1 and lectures of Bio322. <a href="https://www.nmbu.no/course/BIO322" class="uri">https://www.nmbu.no/course/BIO322</a></p>
<p>You can submit your answers on Canvas.</p>
</div>
<div id="task-1-1-explore-the-human-genome-content" class="section level2">
<h2>Task 1-1 Explore the human genome content</h2>
<p>Let’s explore actual genome databases and get to know the genome contents and feature.</p>
<ul>
<li>Go to <a href="https://www.ensembl.org/index.html" class="uri">https://www.ensembl.org/index.html</a></li>
</ul>
<p>Click the following icon. <img src="assets/handson/Slide2.png" alt="Let’s explore the human genome" /></p>
<p>Here, as one example, we are going to look into BRCA2 gene.</p>
<div class="figure">
<img src="assets/handson/Slide4.png" alt="" />
<p class="caption">Investigate the BRCA gene as an example</p>
</div>
<div class="figure">
<img src="assets/handson/Slide5.png" alt="" />
<p class="caption">You can see multiple transcripts of the BRCA2 gene</p>
</div>
<p>Click “Ontologies” and “Phenotypes” to explore the reported function of this gene.</p>
<div class="figure">
<img src="assets/handson/Slide6.png" alt="" />
<p class="caption">Scroll down to see the transcripts and regulartory regions</p>
</div>
<p>Here, there are several questions: ## Question 1 Explore the human genome</p>
<ul>
<li>(1-A) What mechanism generates multiple transcripts from one gene?</li>
<li>(1-B) What is an enhancer, promoter, and transcription factor binding site?</li>
<li>(1-C) When you want to know function of a gene, what research plan would you make?</li>
</ul>
</div>
<div id="task-1-2-explore-multiple-genomes---genome-contents" class="section level2">
<h2>Task 1-2 Explore multiple genomes - genome contents</h2>
<p>Now, we are going to compare human genome and genomes of other animals.</p>
<p>Go back to the human genome page, and click “(i) More information and statistics”</p>
<div class="figure">
<img src="assets/handson/Slide3.png" alt="" />
<p class="caption">Statistics</p>
</div>
</div>
<div id="question-2" class="section level2">
<h2>Question 2</h2>
<ul>
<li>(2-A) How much are the Base Pairs, Coding genes, Non-coding genes, Pseudogenes, Gene transcripts in the human genome?</li>
</ul>
<div id="similarly-lets-explore-the-genome-of" class="section level3">
<h3>Similarly, let’s explore the genome of</h3>
<ul>
<li>Chicken</li>
<li>Common carp</li>
<li>Saccharomyces cerevisiae (What is its common name??)</li>
</ul>
<p>and</p>
<ul>
<li><p>(2-A, continued) write down: Base Pairs, Coding genes, Non-coding genes, Pseudogenes, Gene transcripts in their genomes. If you are ambitious, make a plot of these.</p></li>
<li><p>(2-B) Why is there variation in genome size and gene numbers? How they have been changed in the process of evolution?</p></li>
<li><p>(2-C) How do non-coding genes work?</p></li>
<li><p>(2-D) What are pseudogenes? How are they generated?</p></li>
<li><p>(2-E) Which species has the most genes? What are the advantages of having many genes?</p></li>
</ul>
</div>
</div>
<div id="task-3-explore-multiple-genomes---synteny" class="section level2">
<h2>Task 3 Explore multiple genomes - synteny</h2>
<p>In this task, we are going to compare the structure of multiple genomes.</p>
<p>SynCircos is a tool to compare the contents of chromosomes from mulciple species. <a href="http://bioinfo.konkuk.ac.kr/synteny_portal/" class="uri">http://bioinfo.konkuk.ac.kr/synteny_portal/</a></p>
<p>First, compare chromosome 1 of human, gorilla, mouse and cow.</p>
<div class="figure">
<img src="assets/handson/circos1.png" alt="" />
<p class="caption">four mammals</p>
</div>
<p>We can see that human and gorilla have a quite similar chromosome one content.</p>
<p>However, mouse has only one-third of human chromosome one’s content, and cow’s chromosome one is entirely different from human’s.</p>
<p>So, where in the genome do cow and mouse have human chromosome one’e elements?</p>
<div class="figure">
<img src="assets/handson/circos2.png" alt="" />
<p class="caption">four mammals</p>
</div>
<p>Let’s compare human chromosome one and other chromoosmes of mouse and cow.</p>
<div class="figure">
<img src="assets/handson/circos3.png" alt="" />
<p class="caption">four mammals</p>
</div>
<p>And download the result. <img src="assets/handson/circos4.png" alt="four mammals" /></p>
</div>
<div id="question-3" class="section level2">
<h2>Question 3</h2>
<ul>
<li>(3-B) Where in the genome do cow and mouse have human chromosome one’s elements?</li>
</ul>
</div>
<div id="task-4-explore-multiple-genomes---structure" class="section level2">
<h2>Task 4 Explore multiple genomes - structure</h2>
<p>In this task, we are going to compare the structure of multiple genomes. <a href="http://pipmaker.bx.psu.edu/pipmaker/pip-instr.html" class="uri">http://pipmaker.bx.psu.edu/pipmaker/pip-instr.html</a></p>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
