<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Marie Saitou" />


<title>Bio322.galaxy1</title>

<script src="site_libs/header-attrs-2.10/header-attrs.js"></script>
<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Bio322</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/mariesaitou/Bio322">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Bio322.galaxy1</h1>
<h4 class="author">Marie Saitou</h4>
<h4 class="date">9/25/2021</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-09-27
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Bio322/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20210910code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20210910)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20210910code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20210910)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcommariesaitouBio322treef5142fa39f28da4e4817a74b8dca9813289e37c4targetblankf5142faa"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/mariesaitou/Bio322/tree/f5142fa39f28da4e4817a74b8dca9813289e37c4" target="_blank">f5142fa</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcommariesaitouBio322treef5142fa39f28da4e4817a74b8dca9813289e37c4targetblankf5142faa" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/mariesaitou/Bio322/tree/f5142fa39f28da4e4817a74b8dca9813289e37c4" target="_blank">f5142fa</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .RData
    Ignored:    .Rhistory

Untracked files:
    Untracked:  210922_genome expression_epigenetics.pptx
    Untracked:  BIO322_Teaching plan BIO322 2021.docx
    Untracked:  Bio322.09132021.pdf
    Untracked:  Bio322.09132021.pptx
    Untracked:  Bio322.09152021.backup.pptx
    Untracked:  Bio322.09152021.pdf
    Untracked:  Bio322.09152021.pptx
    Untracked:  Bio322.09202021.pdf
    Untracked:  Bio322.09202021.pptx
    Untracked:  Bio322.09272021.pptx
    Untracked:  Bio322.09272021/
    Untracked:  Bio322scRNAseq.tsv
    Untracked:  Galaxy1-[intestinalData.tsv].tabular
    Untracked:  Galaxy2.txt
    Untracked:  Group.csv
    Untracked:  intestinalData.tsv
    Untracked:  main_workflow.ga
    Untracked:  mouse_intestine_scRNAseq.txt
    Untracked:  scrna_tenx.ga

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/galaxy1.Rmd</code>) and HTML (<code>docs/galaxy1.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mariesaitou/Bio322/blob/f5142fa39f28da4e4817a74b8dca9813289e37c4/analysis/galaxy1.Rmd" target="_blank">f5142fa</a>
</td>
<td>
mariesaitou
</td>
<td>
2021-09-27
</td>
<td>
wflow_publish(“analysis/galaxy1.Rmd”)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mariesaitou/Bio322/f362735ed5a08c91ebe50cd4305b05bef91f8699/docs/galaxy1.html" target="_blank">f362735</a>
</td>
<td>
mariesaitou
</td>
<td>
2021-09-27
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mariesaitou/Bio322/3810aa06d9234fd3fc9a5eb404026a732d965ff8/docs/galaxy1.html" target="_blank">3810aa0</a>
</td>
<td>
mariesaitou
</td>
<td>
2021-09-27
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mariesaitou/Bio322/blob/d777d4d1f95de4e0d4581d5decfea27f1456af71/analysis/galaxy1.Rmd" target="_blank">d777d4d</a>
</td>
<td>
mariesaitou
</td>
<td>
2021-09-27
</td>
<td>
wflow_publish(“analysis/galaxy1.Rmd”)
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="introduction" class="section level2">
<h2>0. Introduction</h2>
<p>This is a set of introductory hands-on exercises to get to know about single-cell RNA sequencing analysis. These exercises align with the learning goals from module 1 and lectures of Bio322. <a href="https://www.nmbu.no/course/BIO322" class="uri">https://www.nmbu.no/course/BIO322</a></p>
<p>In this task, we will learn how to analyze transcriptome data at the single-cell level. (1) We will learn cell clustering based on transcriptome, and how to identify marker genes. (2) Also we will identify multipotent progenitor stem cells in the cell clusters.</p>
<p>Before you start, If you are not familiar with Galaxy, please explore it. (1.1 and 1.2 in <a href="https://mariesaitou.github.io/Bio326/Bio326.genome.html#1_Galaxy_introduction">this page</a>)</p>
<p>We will use dataset from this <a href="https://doi.org/10.1016/j.stem.2016.05.010">paper</a> and pipeline from this: <a href="https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna-raceid/tutorial.html">tutorial</a>. <a href="https://cran.r-project.org/web/packages/RaceID/vignettes/RaceID.html">RaceID</a> is an analytic package for single-cell RNA sequence data sets.</p>
<p>The data was intestinal stem cells of mice, with the aim of discovering distinct cell sub-populations and deriving a lineage tree between them to find out how these sub-populations relate one another.</p>
</div>
<div id="get-the-cell---gene-expression-matrix-data" class="section level2">
<h2>1. Get the cell - gene expression matrix data</h2>
<pre class="bash"><code>https://zenodo.org/record/1511582/files/intestinalData.tsv</code></pre>
<p>Let’s explore the data set. <img src="assets/Bio322.09272021/Slide14.png" /></p>
<p>You can see the data summary and whole data, by clicking the file name and “eye” icon for each. In the dataset, you will see gene names in the first row, and Cell IDs in the first column. The number inside the matrix is the gene expression of each cell.</p>
</div>
<div id="filter-lowly-expressed-genes-and-cells" class="section level2">
<h2>2. Filter lowly-expressed genes and cells</h2>
<p>First, we want to remove low-quality cell and lowly-expressed genes from the dataset with <strong>Initial processing using RaceID Tool</strong>: (You can search for “RaceID” in the left window.)</p>
<p><img src="assets/handson/raceID.png" /></p>
<p>Now, we want to include only:</p>
<ul>
<li>Cells with total transcripts are larger than or equal to <strong>3000</strong>.</li>
<li>Genes that are with more than <strong>5</strong> expression in more than <strong>5</strong> cells.</li>
</ul>
<div id="question" class="section level3">
<h3>Question</h3>
<p>How should we set the parameters?</p>
<details>
<summary>
<strong>Click for answer</strong>
</summary>
<img src="assets/handson/task2.png" />
</details>
<p>In the PDF file (If Galaxy is too slow, you can get the pdf <a href="assets/handson/task2.pdf">here</a>). We will go over the first three graphs.</p>
<ol style="list-style-type: decimal">
<li><p>X-axis: log10 Library Size (total number of transcripts per cell)</p>
<p>Y-axis: Observed number of cells</p></li>
<li><p>X-axis: log10 Feature Set (total number of expressed genes per cell)</p>
<p>Y-axis: Observed number of cells</p></li>
<li><p>X-axis: log10 Library Size after filtering (cells with minimum 3000 transcripts = library size)</p>
<p>Y-axis: Observed number of cells</p></li>
<li><p>Total number of expressed genes per cell after filtering</p></li>
</ol>
</div>
<div id="question-1" class="section level3">
<h3>Question</h3>
<p>How many genes remained? How many cells remained?</p>
<p>If Galaxy is too slow, you can have a look the result below.</p>
<details>
<summary>
<strong>Click</strong>
</summary>
<pre class="bash"><code>- Source:: genes: 20268 , cells: 431
- Filter:: genes: 2089 , cells: 287
         ::  10.3 % of genes remain, 66.6 % of cells remain</code></pre>
</details>
</div>
</div>
<div id="clustering" class="section level2">
<h2>3. Clustering</h2>
<p>Here, we want to <strong>group</strong> cells based on gene expression pattern. I will explain it when most people get the results of <strong>3</strong>..</p>
<p>Execute “Clustering with Race ID” with default setting on the <strong>RDS</strong> data after filtering (2). You can get the RDS file from <a href="assets/handson/RDS.rds">here</a></p>
<p>You will get several result files. Let’s look at the pdf file.</p>
<div id="clustering-method" class="section level3">
<h3>3-1.Clustering method</h3>
<details>
<summary>
<strong>Click for result</strong>
</summary>
<img src="assets/handson/cluster.png" />
</details>
<p>(Top-Left figure) This may look complicated, but important to understand the significance and the algorithm of clustering. The x-axis shows number of clusters. The algorithm explored how many clusters explain the data best (One cluster, two clusters, three clusters…). The lower the y-value is, the cleaner each cluster is.</p>
</div>
<div id="outlier-cells" class="section level3">
<h3>3-2. Outlier cells</h3>
<p>In the pdf, page 2 bottom, you see colorful bars.Each bar represents each cell, and the color is corresponding to cluster. The y-axis shows the probability the cell is outlier (shows distinct gene expression pattern within cluster). You see several bars above the threshold (broken line).</p>
<details>
<summary>
<strong>Click for result</strong>
</summary>
<img src="assets/handson/outlier.png" />
</details>
<p>After outlier detection, several cells were moved to cluster “13” and “14”.</p>
The heatmap shows the similarlity of gene expression pattern between each cell in each cluster.
<details>
<summary>
<strong>Click for result</strong>
</summary>
<img src="assets/handson/heatmap.png" />
</details>
</div>
<div id="marker-genes" class="section level3">
<h3>3-3. Marker genes</h3>
<p>Cluster - Genes per Cluster</p>
<p>The heatmaps below show genes that are specifically expressed in certain cluster. For example, Gstm3, St3gal4, Gna11, Krt8 … show distinct expression pattern from other clusters.</p>
<details>
<summary>
<strong>Click for result</strong>
</summary>
<img src="assets/handson/gene.cluster.png" />
</details>
<p>You can also see a table with all the marker genes at <strong>Cluster - genes per Cluster</strong>. <img src="assets/Bio322.09272021/Slide16.png" /></p>
</div>
<div id="question-2" class="section level3">
<h3>Question</h3>
<p>Examine marker gene heatmap in cluster11. You will see multiple similar genes that are specifically expressed in cluster 11. That is the function of this gene group?</p>
<details>
<summary>
<strong>Click for answer</strong>
</summary>
<p>Defa (Defensine Alpha) is antimicrobial and cytotoxic peptides thought to be involved in host defense. <a href="https://www.omim.org/entry/125220?search=defensin%20alpha&amp;highlight=alpha%20defensin">Ref Link</a></p>
</details>
</div>
</div>
<div id="optional-gene-ontology-analysis" class="section level2">
<h2>(optional) Gene Ontology analysis</h2>
<p>Gene Ontology analysis: given a set of genes, an enrichment analysis will find which terms are over-represented using functional annotations for that gene set.</p>
<p><img src="assets/Bio322.09272021/Slide15.png" /></p>
<p>Let’s see the function of cluster 6 marker genes.</p>
<p>Go to <a href="http://cbl-gorilla.cs.technion.ac.il/">GOrilla</a> and put the following parameters.</p>
<p>Step1: Mus musculus Step2: Two unranked lists of genes (target and background lists)<br />
Step3 Target set: Paste the</p>
<p>gene names of cluster 11 from the table (Cluster - Genes per Cluster) Background set: “upload a file” and put <a href="assets/handson/GO_mouse.txt">this file</a> which includes all the mouse genes. Step4: Select All Step5: “Search Enriched GO terms”</p>
<p>Explore the results… What biological pathways are enriched in cluster 11 marker genes? What about cluster 6 marker genes?</p>
</details>
<details>
<summary>
<strong>Click for answer</strong>
</summary>
<p>Cluster 11 marker genes: immunity-related function Cluster 6 marker genes: lipid-related function</p>
</details>
</div>
<div id="differential-gene-expression-analysis" class="section level2">
<h2>4. Differential gene expression analysis</h2>
<p>I will explain it when most people get the results of <strong>4 and 5</strong>.</p>
<p>Here, we will learn how to visualize and interpret deferentially expressed genes between cell clusters. We can see that cluster 11 looks different from other clusters in both methods.</p>
<p>Now, let’s explore</p>
<p><strong>Cluster Inspection using RaceID</strong> Differential Gene Testing: Yes</p>
<p><strong>Cells in Set A</strong> Name of Set: Cells in 1 Selection method: Cluster Numbers List of clusters: 1</p>
<p><strong>Cells in Set B</strong> Name of Set: Cells in 2 Selection method: Cluster Numbers List of clusters: 2 Use Defaults?: Yes</p>
<p>You will get “MA plot” in the pdf file. Here is how to interpret it. <img src="assets/Bio322.09272021/Slide17.png" /></p>
</div>
<div id="trajectory-analysis" class="section level2">
<h2>5. Trajectory analysis</h2>
<p>Now, we want to know which cell cluster is the multipotent progenitor stem cells. Linage computation using StemID PDF report, page 1.</p>
<details>
<summary>
<strong>Click for result</strong>
</summary>
<img src="assets/handson/network.png" />
</details>
<ol style="list-style-type: decimal">
<li>The first panel: the number of links between each cluster and another cluster.</li>
<li>The second panel: the “Delta-Entropy” which measures the variability of gene expression values within a cluster</li>
<li>The third panel: the number of links multiplied by the gene expression variability</li>
</ol>
<p>Overall, cluster 3 is likely to be the multipotent progenitor stem cells with many links and hich gene expression variability.</p>
<p>Linage computation using StemID PDF report, page 3. You will see connection between each cluster.</p>
<details>
<summary>
<strong>Click for result</strong>
</summary>
<img src="assets/handson/lineage.png" />
</details>
<p>Blue: Higher level of cluster entropy - progenitor cell type Red: Lower level of cluster entropy - mature types Link Width - the number of cells in the cluster Red: Stronger link level Green: Weaker link level</p>
<p>Cluster 11, 10 and 9 seems to be mature cells, while 2 and 3 seems to be progenitor cells. Cells in cluster 3 have potential to be differentiated to cells in cluster 1, 6, 5, 2, 8 and 4.</p>
<p>Well done!!!</p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.0.2 (2020-06-22)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS  10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.0/Resources/lib/libRblas.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.0/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] workflowr_1.6.2

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.7       whisker_0.4      knitr_1.34       magrittr_2.0.1  
 [5] R6_2.5.1         rlang_0.4.11     fastmap_1.1.0    fansi_0.5.0     
 [9] stringr_1.4.0    tools_4.0.2      xfun_0.25        utf8_1.2.2      
[13] git2r_0.28.0     jquerylib_0.1.4  htmltools_0.5.2  ellipsis_0.3.2  
[17] rprojroot_2.0.2  yaml_2.2.1       digest_0.6.27    tibble_3.1.4    
[21] lifecycle_1.0.0  crayon_1.4.1     later_1.3.0      sass_0.4.0      
[25] vctrs_0.3.8      promises_1.2.0.1 fs_1.5.0         glue_1.4.2      
[29] evaluate_0.14    rmarkdown_2.10   stringi_1.7.4    bslib_0.3.0     
[33] compiler_4.0.2   pillar_1.6.2     jsonlite_1.7.2   httpuv_1.6.3    
[37] pkgconfig_2.0.3 </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
